crowdsec-docs/unversioned/user_guides/multiserver_setup.md

```diff
@@ -4,59 +4,158 @@ title: About multi-server setup
 sidebar_position: 10
 ---
 
-Crowdsec's [architecture](/docs/intro#architecture) allows distributed setups, as most components communicate via [HTTP API](/docs/local_api/intro).
+import useBaseUrl from "@docusaurus/useBaseUrl"
+
+## Introduction
+
+Crowdsec's [architecture](/docs/next/intro#architecture) allows distributed setups, as most components communicate via [HTTP API](/docs/next/local_api/intro).
 
 When doing such, a few considerations must be kept in mind to understand the role of each component:
- - The agent is in charge of [processing the logs](/docs/parsers/intro), matching them against [scenarios](/docs/scenarios/intro), and sending the resulting alerts to the [local API](/docs/local_api/intro)
+ - The log processor is in charge of [processing the logs](/docs/next/parsers/intro), matching them against [scenarios](/docs/next/scenarios/intro), and sending the resulting alerts to the [local API](/docs/next/local_api/intro)
  - The local API (LAPI from now on) receives the alerts and converts them into decisions based on your profile
- - The bouncer(s) query the LAPI to receive the decisions to be applied
+ - LAPI also takes care of communication with [CAPI](/docs/next/central_api/intro) to pull blocklists and push alerts to the console.
+ - The remediation component query the LAPI to receive the decisions to be applied
```

Comment:
can we link "decisions" to the decisions definition in the concept (and at the same time, review the decisions definition because I'm not sure it is right)